### DISCUSSION TITLE
Chat to pop up when page load

### DISCUSSION THREAD

**Discussion by caleonte92** [FlowiseAI/Flowise#1246] [Q&A]
Hi, I have embedded my chat onto my website, but I would like to have it pop up(open) when the page loads.
I have searched through the docs and discussions, but no luck on finding anything.
How could I achieve this ?
THanks

**Comment by Chalk4711** [comment]
I am also looking for this. Can anyone help us ?

**Comment by kientv** [comment]
It's cool feature . I think we can do that by JavaScript at document ready event .

**Comment by HenryHengZJ** [comment]
definitely open for any contribution on https://github.com/FlowiseAI/FlowiseChatEmbed !

**Comment by llattarini1982** [comment]
Did u implement it?
